src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -319,7 +330,7 @@ public Transaction(Address sender, BigInteger fee, BigInteger firstValid, BigInt
     public static Transaction createKeyRegistrationTransaction(Address sender, BigInteger fee, BigInteger firstValid,
                                                                BigInteger lastValid, byte[] note, String genesisID,
                                                                Digest genesisHash, ParticipationPublicKey votePK,
-                                                               VRFPublicKey vrfPK, BigInteger voteFirst,
+                                                               VRFPublicKey vrfPK, MerkleVerifier sprfpk, BigInteger voteFirst,
```

Comment:
This function change is considered breaking. Since Java allows function overloading, you could make a new one with these argument as well, but it looks like this function is deprecated anyway in favor of `KeyRegistrationTransactionBuilder`, so it would be fine if you didn't change it at all.